### ISSUE TITLE
Cron Daemon Warning

### ISSUE THREAD

**Issue by matzestefan** [redmatrix/redmatrix#381]
> Hoster for the installation: uberspace.de  
Version of Red#Matrix: 2015-07-01.1080R - 1144  
PHP-Version: 5.5.2

My Cron Daemon for Red#Matrix has sent me a notification:
> PHP Warning:  strlen() expects parameter 1 to be string, array given in \<Red#Matrix-Installation-Path\>/include/crypto.php on line 29

The code that is affected:
```php
function pkcs5_pad ($text, $blocksize)
{
    $pad = $blocksize - (strlen($text) % $blocksize);
    return $text . str_repeat(chr($pad), $pad);
}
```

I couldn't found any loose of functionality, but is there still a problem?

**Comment by redmatrix** [comment]
This is the first I've seen that error; and it looks like it's possible to get it if we passed bad data to pkcs5_pad(), but the rarity tells me it's probably not a huge problem and more likely the result of a temporary glitch. I'll keep this issue open in case we see more of it. In that case we'll probably need to trace back the thing that is being encrypted and figure out where it came from and how it got mangled.

**Comment by matzestefan** [comment]
i have analysed which kind of variable it is instead of a string. The result is:
* Its an array with length 4  
```php
count($text) = 4
```
* the first 4 elements (0-3) are empty

**Comment by Steffov** [comment]
I can confirm this issue. The event is located in: "/package/host/localhost/php-5.6.3/bin/php include/poller.php"

**Comment by redmatrix** [comment]
This is possibly fixed in 2656aeaf9cb8482e22055e9c6788fd15

**Comment by Steffov** [comment]
I updated on 14th. Since the update I didn't got any warning by now. 

Looks pretty good. Thanks a lot!
